src/App.css

```diff
@@ -0,0 +1,49 @@
+.App {
+  text-align: center;
+  height: 100%;
+
+  /* background: #cf4278;
+  background-image: radial-gradient(rgba(0, 0, 0, 0.205) 1px, transparent 0);
+  background-size: 40px 40px;
+  background-position: -19px -19px; */
```

Comment:
- [ ] KIndly ensure you dont leave commented code in your codebase so that your code can be clean and readable